hooks/pre-commit

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+# pre-commit hook for git which builds the docker image for a given cloud-sdk version.
+# Copy into .git/hooks/  folder
```

Comment:
I bet copying this to a particular directory is too much hassle for pretty much everybody.

Can you make this just an `update-version.sh`? See `update.sh` here: https://github.com/docker-library/python